osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -336,23 +336,32 @@ protected sealed override void SkinChanged(ISkinSource skin, bool allowFallback)
         {
             base.SkinChanged(skin, allowFallback);
 
-            updateAccentColour();
+            updateComboColour();
 
             ApplySkin(skin, allowFallback);
 
             if (IsLoaded)
                 updateState(State.Value, true);
         }
 
-        private void updateAccentColour()
+        private void updateComboColour()
         {
             if (HitObject is IHasComboInformation combo)
             {
                 var comboColours = CurrentSkin.GetConfig<GlobalSkinColours, IReadOnlyList<Color4>>(GlobalSkinColours.ComboColours)?.Value;
-                AccentColour.Value = comboColours?.Count > 0 ? comboColours[combo.ComboIndex % comboColours.Count] : Color4.White;
+                UpdateComboColour(comboColours?.Count > 0 ? comboColours[combo.ComboIndex % comboColours.Count] : Color4.White, comboColours);
```

Comment:
The fallback to `Color4.White` in the case of null/no combo colours could be a problem if someone wanted to provide own colours only in those cases. Doing this makes it so "there are no colours to suggest" and "the colour white is suggested" are indistinguishable to the ruleset developer. Would go nullable there.

Reply:
sounds fine too, will wait for confirmation from @smoogipoo.

Reply:
Agree with making nullable.